### PR TITLE
[APM] rename RUM agent github repo

### PR DESCRIPTION
**Pull request by jahtalab**

### CODE REVIEW

**Comment by jahtalab** [comment]
Thanks @bmorelli25 ! This PR only fixes the Github repo name change. I think we should merge this one and change the prefix prefix once the redirects are in place to avoid any 404s in a separate PR.

**Comment by jahtalab** [comment]
@bmorelli25 , I've also added a new branch and updated the current branch.

**Comment by bmorelli25** [comment]
Sounds good. This morning is the 6.6.2 release, so we should hold of on merging this until afterwards. If you'd like, I can merge it this afternoon (PDT)

File: conf.yaml [hunk 1]
```diff
@@ -11,7 +11,7 @@ repos:
     apm-agent-nodejs:     https://github.com/elastic/apm-agent-nodejs.git
     apm-agent-python:     https://github.com/elastic/apm-agent-python.git
     apm-agent-ruby:       https://github.com/elastic/apm-agent-ruby.git
-    apm-agent-js-base:    https://github.com/elastic/apm-agent-js-base.git
+    apm-agent-rum-js:    https://github.com/elastic/apm-agent-rum-js.git
```
Comment (bmorelli25) [inline conf.yaml]:
```suggestion
    apm-agent-rum-js:     https://github.com/elastic/apm-agent-rum-js.git
```

**Pull request merged**